Testscripts/Linux/utils.sh

```diff
@@ -3559,4 +3559,35 @@ function Run_SSHCommand()
 			ssh "${ip}" "${cmd}"
 		fi
 	done
+}
+
+function Check_lsvmbus()
+{
+	# check if lsvmbus exists, or the running kernel does not match installed version of linux-tools
+	lsvmbus_path=$(which lsvmbus)
+	if [[ -z "$lsvmbus_path" ]] || ! $lsvmbus_path > /dev/null 2>&1; then
+		install_package wget
+		wget https://raw.githubusercontent.com/torvalds/linux/master/tools/hv/lsvmbus
+		chmod +x lsvmbus
+		if [[ "$DISTRO" =~ "coreos" ]]; then
```

Comment:
This could be legacy code. Can we re-visit the following
1. Why 2 condition is required for storing lsvmbus ?
2. Do we need extra mv command ? We can use ```wget with -O option``` to download file at correct location.